leveler/commands/lvlset/profile.py

```diff
@@ -57,7 +58,7 @@ async def profilecolors(self, ctx, section: str, color: Union[discord.Color, str
 
         # get correct color choice
         if color == "auto":
-            if not all(lib in globals().keys() for lib in ["numpy", "cluster"]):
+            if not all(module in modules for module in ("numpy", "scipy")):
```

Comment:
```suggestion
            if not all(module in modules for module in ("numpy", "scipy.cluster")):
```
`cluster` may, *or may not* depend on some others packages, and, iirc, not load even if scipy itself is loaded.